README.md

```diff
@@ -32,6 +37,16 @@ that supports Maven to create the Maven project. Add this dependency in the
     </dependency>
 ```
 
+If you are using Gradle for building your functions, you can define the Functions
```

Comment:
"to build" reads a little better than "for building".

Reply:
OK